crypto/CMakeLists.txt

```diff
@@ -78,7 +78,7 @@ if (${ARCH} STREQUAL "arm" OR "${REAL_ARCH}" STREQUAL "arm")
   )
 endif()
 
-if (${ARCH} STREQUAL "aarch64")
+if (${ARCH} STREQUAL "aarch64" OR "${REAL_ARCH}" STREQUAL "aarch64")
```

Comment:
Does this affect iOS?

Reply:
Perhaps. Since we have `OPENSSL_NO_ASM` defined in `boringssl/CMakeLists.txt` for both Android and iOS, which causes `boringssl/src/CMakeLists.txt` to set `REAL_ARCH` to `aarch64` (I suppose for arm64) and `ARCH` to `generic`.

The final effect here is that we will also be compiling `cpu-arm.c` for iOS-arm64 as well (not the case before), which could be a good thing as that brings in ARM/NEON-specific optimizations as well.

Reply:
OK, thanks. We should have Bryan run a sanity check on iOS to make sure nothing is messed up.

I'll add a task.